Userland/Applications/PixelPaint/Layers/Layer.h

```diff
@@ -83,6 +83,7 @@ class Layer
     EditMode edit_mode() { return m_edit_mode; }
     void set_edit_mode(EditMode mode);
 
+    virtual bool is_current_bitmap_editable() { return true; }
```

Comment:
This feels a bit weirdly named. Could it be just `is_editable()`?
Also, what's the relationship between this editability and the `EditMode`?